pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerTimer.java

```diff
@@ -29,7 +29,9 @@
   // metric tracking the freshness lag for consuming segments
   FRESHNESS_LAG_MS("freshnessLagMs", false),
 
-  NETTY_CONNECTION_SEND_RESPONSE_LATENCY("nettyConnection", true),
+  // The latency of sending the response from server to broker
+  NETTY_CONNECTION_SEND_RESPONSE_LATENCY("nettyConnection", false),
```

Comment:
I hope this is backward compatible change since name is not getting changed ?

Reply:
Yes, in fact the boolean isn't get used anywhere. So it's backward compatible.